source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,187 @@
+=================
+Change a Document
+=================
+
+.. default-domain:: mongodb
+
+In this page, you can learn how to change documents in a MongoDB collection
+using two distinct operation types: 
+
+- :ref:`Update <update-operation>`
+- :ref:`Replace <replace-operation>`
+
+Update operations specify the fields and values to change in one or more
+documents. A replace operation specifies the fields and values to replace
+a single document from your collection. 
+
+In the following examples, a paint store sells five different
+colors of paint. The ``paint_inventory`` collection represents their
+current inventory: 
+
+.. code-block:: json
+
+    { "_id": 1, "color": "red", "qty": 5 }
+    { "_id": 2, "color": "purple", "qty": 8 }
+    { "_id": 3, "color": "yellow", "qty": 0 }
+    { "_id": 4, "color": "green", "qty": 6 }
+    { "_id": 5, "color": "pink", "qty": 0 }
+
+.. _update-operation:
+
+Update
+------
+
+Update operations can change the fields and values. They apply changes
+specified in an update document to one or more documents that match your
+query filter. 
+
+The :java-docs:`updateOne() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateOne(org.bson.conversions.Bson,org.bson.conversions.Bson)>` 
+method changes the first document your query filter matches and the
+:java-docs:`updateMany() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateMany(org.bson.conversions.Bson,org.bson.conversions.Bson)>` 
+method changes all the documents your query filter matches. 
+
+You can call the ``updateOne()`` and ``updateMany()`` methods on a
+``MongoCollection`` instance as follows: 
+
+.. code-block:: java
+
+    collection.updateOne(query, updateDocument);
+
+    collection.updateMany(query, updateDocument);
+    
+Update Operation Parameters:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- ``query`` specifies a query filter with the criteria to match documents to update in your collection
+- ``updateDocument`` specifies the fields and values to change in the matching document or documents. For this example, we use the :doc:`Updates builder </fundamentals/builders/updates>` to create the update document.
+
+You can create the ``updateDocument`` using an ``Updates`` builder as
+follows:
+
+.. code-block:: java
+
+    Bson updateDocument = Updates.<update operator>("<field>", "<value>");
```

Comment:
It's confusing to show this as a Java code block, since `<SomeType>` is Java syntax for generics, so this use of `<..>` looks like a generic type instead of a placeholder for a method name.  Maybe just describe it instead?

Reply:
Better, but still confusing because of <update operator>. I suggest just describing this in prose here, since we show an example right below, and there's a link to the Updates builders page as well.

Reply:
I still think it's confusing because now it looks like `operator` is an actual method on `Updates`.  But I'll let it go.